pyproject.toml

```
@@ -1,56 +1,107 @@
 [build-system]
-requires = [ "setuptools>=64", "setuptools_scm>=8" ]
+requires = ["setuptools>=64", "setuptools_scm>=8"]
 build-backend = "setuptools.build_meta"
 
 [project]
 name = "apepay"
-dynamic = [ "version" ]
+dynamic = ["version"]
 description = "Python SDK for ApePay"
-authors = [
-  {name = "ApeWorX LTD", email = "admin@apeworx.io"},
-]
+authors = [{ name = "ApeWorX LTD", email = "admin@apeworx.io" }]
 license = { text = "Apache 2.0" }
 readme = "README.md"
 requires-python = ">=3.10,<4"
 
-dependencies = [
-    "eth-ape>=0.8,<1",
-    "pydantic>=2.7,<3",
-    # NOTE: Pinning issue w/ Ape
-    "eth-typing<5",
-]
+dependencies = ["eth-ape>=0.8.24,<1", "pydantic>=2.7,<3"]
 
 [project.optional-dependencies]
-bot = [
-    "silverback>=0.5,<1",
-]
+bot = ["silverback>=0.5,<1"]
 lint = [
-    "flake8",
-    "black",
-    "isort",
-    "mypy",
-    # NOTE: Be able to lint our silverback add-ons
-    "apepay[bot]"
-]
-test = [
-    "ape-foundry",
-]
-dev = [
-    "apepay[bot,lint,test]"
+  "flake8",
+  "black",
+  "isort",
+  "mypy",
+  # NOTE: Be able to lint our silverback add-ons
+  "apepay[bot]",
 ]
+test = ["ape-foundry"]
+dev = ["apepay[bot,lint,test]"]
 
 [tool.setuptools.packages.find]
-where = [ "sdk/py" ]
+where = ["sdk/py"]
 
 [tool.setuptools.package-data]
-apepay = [ "manifest.json", "py.typed" ]
+apepay = ["manifest.json", "py.typed"]
 
 [tool.setuptools_scm]
 # NOTE: Config entry needed for this plugin to function
 
+[tool.ape]
+name = "ApePay"
+
+[[tool.ape.plugins]]
+name = "arbitrum"
+
+[[tool.ape.plugins]]
+name = "optimism"
+
+[[tool.ape.plugins]]
+name = "foundry"
+
+[[tool.ape.plugins]]
+name = "vyper"
```

Feel like this reads nicer
```suggestion
plugins = [
    { name = "arbitrum" },
    { name = "optimism" },
    { name = "foundry" },
    { name = "vyper" },
]
```

---

1f89aeb